### DISCUSSION TITLE
Send webhook failed

### DISCUSSION THREAD

**Discussion by rapuriteja** [soranoo/TradingView-Free-Webhook-Alerts#49] [Q&A]
ERROR
Send webhook failed.
Reason: {"code":-1,"reason":"","message":"invalid params","data":null}
Status: 400

I am getting this error when i tried to conned my Pionex account webhook, but it works fine when i connect it to webhook.site URL 
![image](https://github.com/soranoo/TradingView-Free-Webhook-Alerts/assets/75659508/83cd5c94-a47d-423e-a16e-829a1cb26080)

**Comment by soranoo** [comment]
Can I have the message that you entered inside the tradingview alert?

**Reply by rapuriteja** [reply]
sure,

{"data":{"action":"{{strategy.order.action}}","contracts":"{{strategy.order.contracts}}","position_size":"{{strategy.position_size}}"},"price":"{{close}}","signal_param":"{}","signal_type":"0d5927ed-2da2-4c6b-acc5-e0ac91efd1f5","symbol":"{{ticker}}","time":"{{timenow}}"}

![image](https://github.com/soranoo/TradingView-Free-Webhook-Alerts/assets/75659508/62513b26-a8a8-47e0-a701-5be05a00777a)

**Comment by soranoo** [comment]
Seems it is correct... Sorry, Im no idea what's wrong with it. You may try to regenerate a new message from Pionex.

**Reply by rapuriteja** [reply]
do you know any other free third party service which allows me to connect this trading view webhook to binance for the futures trading
